lib/node-utils.ts

```diff
@@ -477,31 +477,6 @@ export function isRenderFunction(
   });
 }
 
-// TODO: should be removed after v4 is finished
-export function isRenderVariableDeclarator(
```

Comment:
Removed finally. A better equivalent has been created in `detect-testing-library-utils.ts`